### ISSUE TITLE
Web Monetization: add option to append instead of override

### ISSUE THREAD

**Issue by SethFalco** [WeblateOrg/weblate#7915]
### Describe the problem

I noticed you either had or created your own Interledger Payment Pointer and put it in the repo.
I think that's great, but I think it'd be better hosts had the option to opt-in to keep you in the list as well as add their own. 🤔 

### Describe the solution you'd like

Maybe it's worth adding an option to append specified pointers to the list of default ones, instead of overriding them entirely? Then hosts can choose to utilize probabilistic revenue sharing so Weblate continues to get some support from it.

### Describe alternatives you've considered

Technically, someone could just copy and paste any pre-existing default ones into their config before adding their own, but this is a bit cumbersome and won't be updated if the defaults change. (i.e. change in provider)

**Comment by nijel** [comment]
Makes sense. On the other side, I wanted to have the option to fully override or disable this as well.

**Comment by SethFalco** [comment]
Agreed, maybe it'd be nice to make a new option which is separate from the list of pointers.

```
WEB_MONETIZATION_MODE: 'DISABLED' | 'APPEND' | 'OVERRIDE'
```

* `DISABLED` - Disables Web Monetization
* `APPEND` - Effective pointers are the internal pointers, appended with the ones in the config.
* `OVERRIDE` - Effective pointers are the config ones only, which defaults to the internal pointers anyway. (default)

Only reason I made this an issue instead of a PR, is that to do this, there would have to be consts somewhere to save your pointer that isn't the config, and I wasn't sure where was best for that. 🤔 

When it's set to `APPEND`, it should also check an internal pointer is already in the config to avoid giving the internal ones more weight by making them appear in the list more than once.

I can review it on the weekend maybe since it's just config stuff, so it shouldn't be too annoying to do.

**Comment by nijel** [comment]
See 1d093bc2795 for example how such setting was introduced.
